src/Writer/XLSX/RowAttributes.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenSpout\Writer\XLSX;
+
+use OpenSpout\Common\Exception\InvalidArgumentException;
+
+class RowAttributes
+{
+    private bool $visible;
+    private bool $collapsed;
+
+    /** @var null|int<0, 7> */
+    private ?int $outlineLevel;
+
+    /**
+     * @param null|int<0, 7> $outlineLevel
+     */
+    public function __construct(?int $outlineLevel, bool $collapsed = false, bool $visible = true)
```

Comment:
This whole class can just be a Value Object. Let's remove PHP `8.0.0` support and add an Enum for the OutlineLevel, then we can rewrite this class as:
```php
enum OutlineLevel: int
{
    case ONE    = 1;
    case TWO    = 2;
    case THREE  = 3;
    case FOUR   = 4;
    case FIVE   = 5;
    case SIX    = 6;
    case SEVEN  = 7;
}

final class RowAttributes
{
    public function __construct(
        public readonly ?OutlineLevel $OUTLINE_LEVEL = null,
        public readonly bool $COLLAPSED = false,
        public readonly bool $VISIBLE = true,
    ) {}
    
    public function isEmpty(): bool
    {
        return null === $this->OUTLINE_LEVEL
            && false === $this->COLLAPSED
            && true === $this->VISIBLE
        ;
    }
}
```

Reply:
> Let's remove PHP 8.0.0 support

No please, our project still on PHP 8.0)

Reply:
https://github.com/openspout/openspout/blob/4.x/UPGRADE.md#upgrading-from-3x-to-40

I'm gonna do it anyway in 14 days https://www.php.net/supported-versions.php and I won't be able to switch `$OUTLINE_LEVEL` type in a minor release, so let's just create it as an Enum.

Until you are on 8.0 you can download your PR, modify it and apply it with https://github.com/cweagans/composer-patches to the latest `openspout:4.8` release

Reply:
Oh, I see... Unfortunately, I have only PHP 8.0 and cannot do it, sorry.